.github/workflows/run-zombienet-tests.yml

```diff
@@ -70,7 +70,7 @@ jobs:
             if [ "$test_bundle" == "weekly_bundle" ]; then
               tests='["zombie_tanssi_warp_sync", "zombie_tanssi_parathreads", "zombie_tanssi_metrics"]'
             elif [ "$test_bundle" == "manual_bundle" ]; then
-              tests='["zombie_tanssi_relay_finalization_recovery", "zombie_simple_template_offchain", "zombie_tanssi_keep_db", "zombie_tanssi_one_node", "zombie_tanssi_relay_starlight", "zombie_tanssi_relay_unneeded_para", "zombie_tanssi_relay_keystore"]'
+              tests='["zombie_tanssi_relay_finalization_recovery", "zombie_simple_template_offchain", "zombie_tanssi_keep_db", "zombie_tanssi_one_node", "zombie_tanssi_relay_starlight", "zombie_tanssi_relay_unneeded_para", "zombie_tanssi_relay_keystore", "zombie_tanssi_relay_foreign_tokens"]'
```

Comment:
are we added another suite for this? I think we should at least try to run it with the one we have. @Agusrodri I think this could be a candidate to add to your refactor once you have it no?